draft-ietf-oauth-cross-device-security.md

```diff
@@ -233,6 +254,50 @@ Figure: Cross-Device Flow: User-Transferred Authorization Data Pattern
 
 The Authorization Server may choose to authenticate the user before sending the authorization data. The authorization data may be delivered as a text message or through a mobile application.
 
+## Cross-Device Session Transfer
+In a cross-device service transfer flow a user is authenticated and then authorizes the session transfer on one device, referred to as the Authorization Device (e.g. a personal computer, web portal or application), and transfers the session to the device where they will continue to consume the session, referred to as the Consumption Device (e.g. a mobile phone or portable device).
+
+Cross-device session transfer flows enables users to transfer state information, including the authentication and authorization state of the current session to a second device without having to re-enter configuration, authentication or authorization information. This allwos users to seamlessly:
+
+* Add new devices to a network.
+* Onboard to a mobile application.
+* Provision new credentials (e.g. [@OpenID.VCI]).
+
+### Cross-Device Session Transfer Pattern
```

Comment:
```suggestion
### Session Transfer Pattern
```

Reply:
It was only for consistency with the label in the examples (e.g., `Example A5: Add a Device to a Network (Session Transfer Pattern)`), but I see your point.